src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
```

Comment:
you not need to provide `justify-content: flex-end;`, make parent containter flexible and add `justify-content: space-between` for them to set the maximum intend between the logo and the navigation